addons/docs/package.json

```diff
@@ -60,6 +60,7 @@
     "@mdx-js/react": "^1.6.22",
     "@storybook/addons": "6.2.0-alpha.32",
     "@storybook/api": "6.2.0-alpha.32",
+    "@storybook/builder-webpack4": "^6.2.0-alpha.32",
```

Comment:
@gaetanmaisse Why do we need this?

Reply:
Because of this line: https://github.com/storybookjs/storybook/blob/c8cf5a567bb65a9180f30ae1c921093adeb0dc41/addons/docs/src/frameworks/common/preset.ts#L9